docs(configuration) : Add more option to resolve.alias (#3358)
webpack/webpack.js.org: src/content/configuration/resolve.md

### `resolve.alias`

> `object` `[string]` `boolean: false`

Reviewer: `resolve.alias` always takes object I guess.
The changes you are recommending is fit for `resolve.alias[key]`,
cause the changes are recommending

```js
resolve: {
  alias: "string";
  or;
  alias: false;
}
```

Which I think is not a valid.
it should be

```js
resolve: {
  alias: {
    _: string; // or false
  }
}
```

So the description is not describing that. Could you make those changes ?

Reviewer 2: You are completely right, great spot!

Author: An array of strings is possible , so it could be any of object and array of strings 

https://github.com/webpack/webpack/blob/21a80704ed4478fbfb07d1b786738c18b4f194bf/schemas/WebpackOptions.json#L2016

Author: The all confusion was created because of 

https://github.com/webpack/webpack/blob/21a80704ed4478fbfb07d1b786738c18b4f194bf/schemas/WebpackOptions.json#L2022 

`Resolve.alias.alias`

The second alias accepts `string` `bool` or `[string]`

W> `resolve.alias` takes precedence over other module resolutions.

> `resolve.alias` now supports array of `strings` and `false` to ignore a module

Reviewer 2: This needs to be a
```
W> [string] and `false` values are supported since webpack 5.0.x
```

Author: Made apt changes.

```js
module.exports = {
  //...
  resolve: {
    alias: {
      _: [path.resolve(__dirname, 'src/utilities/'), path.resolve(__dirname, 'src/templates/')]
    }
  }
};
```

> 

Reviewer 2: Please add a description of what happens when you set it to `false`

Author: Would this work ?

`resolve.alias` when set to `false` will ignore a module.

W> `resolve.alias` takes precedence over other module resolutions.

> W> [string] and `false` values are supported since webpack 5.0.x.

Reviewer 2: Forgot to to wrap [string] in \`
```suggestion
W> `[string]` and `false` values are supported since webpack 5.0.0
```

```js
module.exports = {
  //...
  resolve: {
    alias: {
      _: [path.resolve(__dirname, 'src/utilities/'), path.resolve(__dirname, 'src/templates/')]
    }
  }
};
```

> `resolve.alias` when set to `false` will ignore a module.

Reviewer 2: I am not really clear from reading it, what exactly happens, webpack won't bundle it? e.g. import will not result in an error and its value will be `undefined`? e.g.
```
import x from './ignored-module';

console.log(x); // undefined
```

Or no?

Also text can be improved:
```suggestion
Setting `resolve.alias` to `false` will tell webpack to ignore a module.
```

W> `resolve.alias` takes precedence over other module resolutions.

> W> [`null-loader`](https://github.com/webpack-contrib/null-loader) is deprecated in `webpack@5`. use `alias: { xyz$: false }` or absolute path `alias: {[path.resolve(__dirname, "....")]: false }`

Reviewer 2: ```suggestion
W> [`null-loader`](https://github.com/webpack-contrib/null-loader) is deprecated in webpack 5. use `alias: { xyz$: false }` or absolute path `alias: {[path.resolve(__dirname, './path/to/module')]: false }`
```

> W> `[string]` values are supported since webpack 5.0.0.

Reviewer 2: It's still in beta, lets just say 5 here too?

Author: Done, Let's get this merged, It has been sitting here for almost 6 months now :)
